pkg/crc/machine/libvirt/driver.go

```diff
@@ -0,0 +1,47 @@
+package libvirt
+
+import (
+	"path/filepath"
+
+
+	"github.com/code-ready/crc/pkg/crc/constants"
+
+	"github.com/code-ready/crc/pkg/crc/machine/config"
+
+	"github.com/docker/machine/libmachine/drivers"
```

Comment:
Shouldn't coming from the docker/machine side.

Reply:
typo